fdbserver/include/fdbserver/EncryptionOpsUtils.h

```diff
@@ -48,4 +48,11 @@ inline bool isEncryptionOpSupported(EncryptOperationType operation_type) {
 	}
 }
 
+inline bool isEncryptionOpSupported(EncryptOperationType operation_type, EncryptionAtRestMode encryptMode) {
```

Comment:
I would move this function to BlobWorker.actor.cpp and eventually delete the EncryptionOpsUtils.h once we remove the encryption knobs.